aip/auth/4111.md

```diff
@@ -74,6 +94,9 @@ file.
 * For the `aud` field, specify the audience. The default audience value
 **should** be `https://[API_ENDPOINT]/`. (e.g. `https://pubsub.googleapis.com/`)
 
+* For the `scope` field, specify the scope. Please note that `aud` and `scope`
```

Comment:
Here it says aud and scope cannot be specified at the same time, yet the example above specifies both. Maybe we should instead say that 'aud' and 'scope" should not be specified at the same time and the behavior is "undefined"  if both are specified? Maybe clients can give preference to 'aud'? (Note the similarity to our other AIP where API_KEY and Credentials should not be specified at the same time, but is still accepted.)

Reply:
I updated the example into two examples.

It is gaia BE team requirement to not accept aud and scope at the same time. The API key case is a bit different since it is sometimes useful to be passed along with the credentials, for example, API key restrictions. In this case, setting both aud and scope seems completely unnecessary.